### DISCUSSION TITLE
remaining BIFs

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#191] [Progress Updates]
I am completing the outstanding conversion bifs  x2c,d2c, ... by tomorrow. There remain a few outstanding ones, which we need to discuss. Most of them are not difficult to implement (min, max) but need clarification on the number input numbers, etc.
We should discuss this in the next call.

**Comment by adesutherland** [comment]
One suggestion - if min(), max() etc. use floats then the compiler will automatically convert integer provided arguments to floats. So not the fastest but a working solution in the meantime ...

**Comment by Peter-Jacob** [comment]
the unknown numbers of input parms is the problem, can be 2 can be 10.
